modules/social_features/social_search/social_search.install

```diff
@@ -194,3 +194,34 @@ function social_search_update_8905() {
     \Drupal::logger('social_search')->info($e->getMessage());
   }
 }
+
+/**
+ * Removed created field for profile and added 'Owner » User » Created' field.
+ */
+function social_search_update_10001() {
+  /** @var \Drupal\update_helper\Updater $updateHelper */
+  $updateHelper = \Drupal::service('update_helper.updater');
+
+  // Execute configuration update definitions with logging of success.
+  $updateHelper->executeUpdate('social_search', 'social_search_update_10001');
+
+  // Output logged messages to related channel of update execution.
+  return $updateHelper->logger()->output();
+}
+
+/**
+ * Update search users with new 'created' field.
+ */
+function social_search_update_10002() {
+  try {
```

Comment:
Hi @navneet0693 

I was thinking, for the sites with high amount of users, maybe it's better to provide an optional setting.
e.g.

`Settings::get('social_user_update_indexes')` so people can set it to FALSE if they dont want an update hook to trigger the re-index but rather do it manually or by the cron.

By default it is set to true so we can be sure it works correctly for those who don't care.

What do you think?

But especially

Reply:
@ronaldtebrake good idea!

Reply:
Added : https://github.com/goalgorilla/open_social/pull/2314/commits/6b0194df00a054ce61c06b5c79a2e4a705deef31